tests/e2e/e2e.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from "@playwright/test";
+
+test("Create an account", async ({ page }) => {
+  await page.goto("http://localhost:3030/login");
```

Comment:
move the  URL to the `playwright.config.ts` use.baseURL. https://playwright.dev/docs/test-configuration

Reply:
I've added the base URL to the config file and updated the tests to use the relative paths.